lib/pavilion/plugins/results/filecheck.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+from pavilion import config
+from pavilion import result_parsers
+from pavilion import utils
+import yaml_config as yc
+import glob
+import os
+import sys
+
+
+class Filecheck(result_parsers.ResultParser):
+    """Checks the working directory for the given file name.
+    The parser will tell the user if the filename exists or not. """
+
+    def __init__(self):
+        super().__init__(name='filecheck',
+                         description="Checks working directory"
+                         "for given file name")
+
+    def get_config_items(self):
+
+        config_items = super().get_config_items()
+        config_items.extend([
+            yc.StrElem('filename', required=True,
+                       help_text="The filename result parser"
+                                 "takes in a string and checks"
+                                 "the working directory for said "
+                                 "filename. It will return True or "
+                                 "False.")
+        ])
+
+        return config_items
+
+    def _check_args(self, filename=None):
+        if filename == "" or filename is None:
+            raise result_parser.ResultParserError(
+                "File name cannot be null"
+            )
+
+    def __call__(self, test, file, filename):
+        try:
+            pav_cfg = config.find()
+        except Exception as err:
+            sys.exit(-1)
```

Comment:
Paul is going to ask you to change this to the ERRNO stuff.  Ask Jordan.

Reply:
Nope, even worse. 

You shouldn't be using sys.exit at all. Let the exception happen, Pavilion will catch it, and error report it.

Reply:
In some situations you do want to catch exceptions and re-raise them with a more meaningful message (that also includes the original message), usually along the lines of:
```
try: 
    stuff...
except (ExpectedException, OtherException) as err:
    raise ModuleSpecificException(
        "This specific bad thing happened in this specific place related to test '{test.id}': {err}"
        .format(test=test, err=err)

You should only catch those specific exceptions you expect, and never the really generic ones like Exception or BaseException.